### PR TITLE
ref: Use @sentry/react

### PR DESCRIPTION
**Pull request by AbhiPrasad**
### Motivation

As we are preparing to add public docs for https://github.com/getsentry/sentry-javascript/tree/master/packages/react, we figured it would be best to dogfood the react package first with sentry.

`@sentry/react` is a superset of the `@sentry/browser` package, with a custom `Profiler` component added, as well as an `ErrorBoundary` component out of the box.

### Implementation

List of changes:
1. Replace all instances of `@sentry/browser` with `@sentry/react`
2. Delete built in profiler, use `withProfiler` from `@sentry/react`

### Future

Another PR to add the `@sentry/react` `ErrorBoundary` component.

### CODE REVIEW

File: src/sentry/static/sentry/app/components/loadingIndicator.tsx [hunk 1]
```diff
@@ -82,4 +80,6 @@ LoadingIndicator.propTypes = {
   hideSpinner: PropTypes.bool,
 };
 
-export default withProfiler(LoadingIndicator);
+export default withProfiler(LoadingIndicator, {
+  hasUpdateSpan: false,
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/loadingIndicator.tsx:84]: mmm I can't form words atm, but `has` doesn't read to me that this is an option for withProfiler... maybe something like `includeUpdateSpan`?

**Review by billyvg — APPROVED** [review]
🚀

Left a comment re: sdk API, but otherwise looks good

File: src/sentry/static/sentry/app/components/loadingIndicator.tsx (hunk 1 above)
Comment (markstory) [reply src/sentry/static/sentry/app/components/loadingIndicator.tsx:84]: Or just `updateSpan`

Comment (billyvg) [reply src/sentry/static/sentry/app/components/loadingIndicator.tsx:84]: `updateSpan` makes it sound like the profiler will perform an update of a span -- maybe `includeUpdates`?

Comment (AbhiPrasad) [reply src/sentry/static/sentry/app/components/loadingIndicator.tsx:84]: Something like
```
// If time component is on page should be displayed as a span. True by default.
includeRender?: boolean;
// If component updates should be displayed as spans. True by default.
includeUpdates?: boolean;
```

Comment (AbhiPrasad) [reply src/sentry/static/sentry/app/components/loadingIndicator.tsx:84]: For now I am going to merge this in, but I will make the API changes and come back and update.

**Pull request merged**